src/main/java/com/teragrep/cfe_39/consumers/kafka/RecordOffset.java

```diff
@@ -0,0 +1,59 @@
+/*
+   HDFS Data Ingestion for PTH_06 use CFE-39
+   Copyright (C) 2022  Fail-Safe IT Solutions Oy
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
+
+package com.teragrep.cfe_39.consumers.kafka;
+
+// This is the class for handling the Kafka record topic/partition/offset data that are required for HDFS storage.
+public class RecordOffset extends Offset{
```

Comment:
implements Offset

Reply:
Solved  in commit 70c6f26b298d4bfd93b2e8e0cdf47f5f4c5ce7c3 and c1c455cba47794cdf3379116ef2932bf61a22128